Jenkinsfile

```diff
@@ -128,6 +128,22 @@ pipeline {
         }
         */
 
+        stage('scons build test') {
+          agent {
+            dockerfile {
+              filename 'Dockerfile.openpilot_base'
+              args '--user=root'
+            }
+          }
+          steps {
+            sh "git config --global --add safe.directory '*'"
+            sh "git submodule update --init --depth=1 --recursive"
+            sh "git lfs pull"
+            sh "scons --clean && scons --no-cache --random -j42"
+            sh "scons --clean && scons --no-cache --random -j42"
```

Comment:
how about we do two passes: one normal build with lots of cores and add random to the second one?